airflow/models/baseoperator.py

```diff
@@ -646,6 +646,12 @@ def render_template_from_field(self, attr, content, context, jinja_env):
         rt = self.render_template
         if isinstance(content, str):
             result = jinja_env.from_string(content).render(**context)
+        elif isinstance(content, tuple):
+            if type(content) is not tuple:
+                # Special case for named tuples
+                result = content.__class__(*(rt(attr, e, context) for e in content))
+            else:
+                result = (rt(attr, e, context) for e in content)
         elif isinstance(content, (list, tuple)):
```

Comment:
```suggestion
        elif isinstance(content, list):
```

It can't be a tuple anymore - that is handled by the previous elif you've just added

Reply:
Oops, good catch! Let me fix that.